Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -65,6 +66,10 @@ public class JavaScriptLifter: ComponentBase, Lifter {
         func expr(for v: Variable) -> Expression {
             return expressions[v] ?? Identifier.new(v.identifier)
         }
+
+        if options.contains(.collectTypes) {
+            w.emitBlock(initializeTypeCollection)
```

Comment:
Maybe we should make it more obvious that these constants contain JavaScript code as string, maybe simply by prefixing them with `JS` or `JSCode` or so?

Reply:
I would go with suffix, e.g. `initTypeCollectionScript`?

Reply:
SGTM!